src/plugins/instrument_xhr.js

```diff
@@ -234,6 +248,29 @@ class InstrumentXHR {
         }
     }
 
+    _getXHRSpan(xhr) {
+        var self = this;
+
+        // Check if we've already joined successfully; if so, return early.
+        if (xhr.__xhr_span) {
```

Comment:
the first take on this (which didn't make it to `lightstep-tracer-javascript`) didn't distinguish between the actual XHR object (`this` in the various functions *returned* by `_instrument*()`) and the interposition class, `InstrumentXHR`. That meant that a lot of things were broken, but at first glance they didn't seem to be since the previously-seen XHR would have its tracer state applied to the next and we happened to be looking at serialized requests.